lib/devices/set_system_date_and_time.ex

```diff
@@ -23,7 +30,13 @@ defmodule Onvif.Devices.SetSystemDateAndTime do
             element(:"tt:TZ", system_date_time.time_zone.tz )
           ]
         ),
+        List.flatten([utc_date_time_element(system_date_time, set_time?)])
+      ])
+    ])
+  end
 
+  def utc_date_time_element(system_date_time, false) do [] end
```

Comment:
One liners don't need the `end`, but they need comma and colon:

```suggestion
  def utc_date_time_element(system_date_time, false), do: []
```

Reply:
done